cmd/brew-postgresql-upgrade-database.rb

```diff
@@ -49,6 +49,13 @@
   # https://www.postgresql.org/docs/10/static/pgupgrade.html
   ohai "Upgrading #{name} data from #{pg_version_data} to #{pg_version_installed}..."
 
+  # get 'lc_collate' from old DB"
+  unless system "#{old_bin}/pg_ctl", "-w", "-D", datadir, "status", out: File::NULL
+    safe_system "#{old_bin}/pg_ctl", "-w", "-D", datadir, "start", out: File::NULL
```

Comment:
Skip the `out: File::NULL` and use `brew services` here

Reply:
Here I need to use `old_bin` with new `datadir`. Can I specify `datadir` when I use `brew services`?

Reply:
`old_bin` will always relate to an installed versioned formula e.g. `postgresql@9.5` rather than an old Cellar directory so you shouldn't need to specify it.

Reply:
Sorry, I still can't get it. `brew services run postgresql@9.6` uses data in `/usr/local/var/postgresql@9.6`, but the data must be used in `/usr/local/var/postgres`. 

How can I run the service?

Reply:
Sorry, I see, yes you'll need to avoid `brew services` in this case then.  You'll have to run this after the `brew services stop` below to avoid clashing with a running version.

Reply:
I see. I moved the part after `brew services stop`. Thanks for advice.